recipes/server_install.rb

```diff
@@ -182,6 +188,17 @@
   notifies (node['rundeck']['restart_on_config_change'] ? :restart : :nothing), 'service[rundeck]', :delayed
 end
 
+template "/etc/init/rundeckd.conf" do
```

Comment:
The rundeck upstart config that comes with rundeck had a bug (https://github.com/rundeck/rundeck/pull/2246) . Thats why I added the rundeck upstart config. But it seems they fixed it in latest rundeck version. I shall remove that.